elixir/lib/my_app/router.ex

```diff
@@ -0,0 +1,12 @@
+defmodule MyApp.Router do
+  import Plug.Conn
+
+  def init(opts), do: opts
+
+  def call(%Plug.Conn{request_path: "/_ah/health"} = conn, _opts) do
+    send_resp(conn, 200, "👌")
```

Comment:
This slipped past in the Swift PR #9 😁. Official docs do explicitly state that the value of the message body is [ignored](https://cloud.google.com/appengine/docs/flexible/go/how-instances-are-managed) though.

> You can write your own custom health-checking code. It should reply to `/_ah/health` requests with a HTTP status code 200. The response must include a message body, however, the value of the body is ignored (it can be empty).